packages/components/src/breadcrumbs/test/index.tsx

```diff
@@ -109,4 +109,30 @@ describe( 'Breadcrumbs', () => {
 			screen.queryByRole( 'button', { name: 'More breadcrumb items' } )
 		).not.toBeInTheDocument();
 	} );
+
+	it( 'renders a custom item link', async () => {
```

Comment:
To make this test more comprehensive, we could also:

- check that the render prop is not applied to the last item;
- check that the render prop is applied to the menu items when `variant="compact"`